tests/test_svg.py

```diff
@@ -70,7 +73,45 @@ def test_multiple_closed_paths(self):
              ('L', [100.0, 60.0]),
              ('Z', [])])
 
-    def test_moveto_command(self):
+
+class MoveToCommandTC(SvgParserBaseTC):
+
+    def test_moveto_absolute(self):
```

Comment:
I don't mind to have `_moveto_` in the test function names, but they are redundant.

Reply:
If you don't have strong opion to remove it, I would like to keep it for the ease of keyword searching.